docs/UserGuide.md

```diff
@@ -410,11 +430,13 @@ This feature allows you to find all tasks assigned to a single member.
 
 Format: `findTasksFor NAME`
 
-* The search using NAME is case-sensitive. e.g. `David Li` will only match with `David Li`.
+* The search using NAME is case-sensitive.
+  * e.g. `David Li` will only match with `David Li`.
 * NAME must match an entire member's name exactly in order for the tasks assigned to that member to be found. 
-  
+  * e.g. `David` will not match with `David Li`
```

Comment:
For this part, if we are going to give examples for case-sensitive & matching exactly, should we put it for editMember/deleteMember etc. too?

Reply:
I think no need. I put this because most of our other search functions are case insensitive and wanted to make it clear how its different from the other find functions